pyvista/themes.py

```diff
@@ -1165,9 +1165,17 @@ class _SliderConfig(_ThemeConfig):
     >>> import pyvista as pv
     >>> pv.global_theme.slider_styles.classic.slider_length = 0.02
     >>> pv.global_theme.slider_styles.classic.slider_width = 0.04
-    >>> pv.global_theme.slider_styles.classic.slider_color = (0.5, 0.5, 0.5)
+    >>> pv.global_theme.slider_styles.classic.slider_color = (
```

Comment:
If we do something like the below, it is less messy formatting.

```py
    >>> slider_styles = pv.global_theme.slider_styles
    >>> slider_styles.classic.slider_color = (0.5, 0.5, 0.5)
```

Reply:
Fixed in 062fe1ab41418178050ae4121a75a2442a22f5f7.